tests/lock_file/CMakeLists.txt

```diff
@@ -15,26 +15,36 @@
 
 file(GLOB PACKAGE_LOCK_TESTS "*_test")
 
+include(fail.cmake)
+
 foreach(dir ${PACKAGE_LOCK_TESTS})
   file(RELATIVE_PATH test_prefix ${TOIT_SDK_SOURCE_DIR} ${dir})
   set(test1 "${test_prefix}/test.toit")
-  add_test(
-    NAME ${test1}
-    COMMAND $<TARGET_FILE:toit.run> ${test1}
-    WORKING_DIRECTORY ${TOIT_SDK_SOURCE_DIR}
-    )
-  set_tests_properties(${test1} PROPERTIES TIMEOUT 40)
+
+  if (NOT "${test1}" IN_LIST TOIT_SKIP_TESTS)
```

Comment:
Maybe try to be consistent about using continue for skipping?

Reply:
I could do a `continue` for the `test2` but not for `test1` as you could skip the first, but not the second.
I think it's more consistent to have both in the if instead.